code/game/machinery/recharger.dm

```diff
@@ -3,7 +3,7 @@
 	icon = 'icons/obj/stationobjs.dmi'
 	icon_state = "recharger"
 	base_icon_state = "recharger"
-	desc = "A charging dock for energy based weaponry."
+	desc = "A charging dock for energy based weaponry and PDAs."
```

Comment:
```suggestion
	desc = "A charging dock for energy based weaponry, PDAs, and other devices."
```

Maybe just kill 2 birds with one stone here? They can charge the tagger I believe and there's probably others as well.

Reply:
What tagger do they recharge?

Reply:
it charges PDAs, tablets, batons, energy weapons and energy magazines so sure

Reply:
Huh, I could've sworn destination taggers used cells and needed to be recharged. Shows how much I play cargo.